pyproject.toml

```diff
@@ -12,13 +12,13 @@ authors = [
 license = "MIT"
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = ">=3.7,<3.12"
```

Comment:
Updating click drops py3.6 support. The project also doesn't work with py3.12 as per test run as-is, hence the updated version range. We can always revisit as necessary.